src-theme/src/routes/clickgui/setting/TogglableSetting.svelte

```diff
@@ -42,55 +43,58 @@
 <div class="setting">
     {#if nestedSettings.length > 0}
         <!-- svelte-ignore a11y-no-static-element-interactions -->
-        <div class="head expand" class:expanded on:contextmenu|preventDefault={toggleExpanded}>
+        <div class="head expand" class:expanded on:contextmenu|preventDefault={toggleExpanded}
+             class:animate={$animationsEnabled}>
             <Switch
-                name={$spaceSeperatedNames ? convertToSpacedString(cSetting.name) : cSetting.name}
-                bind:value={enabledSetting.value}
-                on:change={handleChange}
+                    name={$spaceSeperatedNames ? convertToSpacedString(cSetting.name) : cSetting.name}
+                    bind:value={enabledSetting.value}
+                    on:change={handleChange}
             />
-            <ExpandArrow bind:expanded on:click={() => skipAnimationDelay = true} />
+            <ExpandArrow bind:expanded on:click={() => skipAnimationDelay = true}/>
         </div>
     {:else}
-        <div class="head" class:expanded>
+        <div class="head" class:expanded class:animate={$animationsEnabled}>
             <Switch
-                name={$spaceSeperatedNames ? convertToSpacedString(cSetting.name) : cSetting.name}
-                bind:value={enabledSetting.value}
-                on:change={handleChange}
+                    name={$spaceSeperatedNames ? convertToSpacedString(cSetting.name) : cSetting.name}
+                    bind:value={enabledSetting.value}
+                    on:change={handleChange}
```

Comment:
```suggestion
                name={$spaceSeperatedNames ? convertToSpacedString(cSetting.name) : cSetting.name}
                bind:value={enabledSetting.value}
                on:change={handleChange}
```
Not sure why that extra space is needed...

Reply:
code styling

Reply:
No? It was already indented enough. Usually you do
```jsx
<x
    prop1="value"
    prop2="value"
    prop3="value"
    prop4="value">
```
instead of adding an extra 4 spaces.